keylime/tenant.py

```diff
@@ -949,8 +961,7 @@ def do_quote(self):
                     do_quote = RequestsClient(cloudagent_base_url, tls_enabled=False)
                     response = do_quote.get(params, timeout=self.request_timeout)
 
-                print(response)
-                response_body = response.json()
+                response_json = Tenant._jsonify_response(response, False, True)
```

Comment:
Here: no more printing ?

Reply:
Correct, no more printing here

Reply:
Trying to keep it behaving as it was before

Reply:
was before printing: `print(response)`

Reply:
ah, very true... I was assuming this was a "leftover print", but you're right. Absent of clear evidence that it is indeed the case, we should retain the print out functionality